### ISSUE TITLE
Deepsort tracking uses all the CPU cores to almost 100 %

### ISSUE THREAD

**Issue by fareed945** [nwojke/deep_sort#237]
Hey a clarification!
while running detection over a video, I see that my entire CPU memory is being used. I'm not able to run it on multiple threads as it leads to slowness.
Did anyone face this issue ?
Any help would be appreciated

**Comment by fareed945** [comment]
@nwojke

**Comment by rajavi-cis** [comment]
Yes, this same issue is also faced my me. Please Help us with some suggestion.

**Comment by RomeroBarata** [comment]
Hi, I'm having the same issue and the best solution I've found so far is to use `taskset` as described in here https://pretagteam.com/question/how-to-limit-number-of-cpus-used-by-a-python-script-wo-terminal-or-multiprocessing-library

In summary `taskset --cpu-list 0,1,2,3 python deep_sort_app.py ...`.

This is an old issue so I hope this solution may serve as future reference.

Kind regards,
Romero
